.github/workflows/docker-image.yml

```diff
@@ -0,0 +1,41 @@
+name: Docker Image CI
+
+on:
+  push:
+    branches: [ "main" ]
```

Comment:
maybe just on tags like `v124`? because we may put bad code to the main branch

Reply:
Yes, you can.

But it's hard to debug if I do not have the repo permission.
I have no permission to push images to your GHCR instance.
Github action has strict permission.

I recommend you copy the action from [my existing file](https://github.com/gocrane/crane/blob/main/.github/workflows/build-images.yml).

It can auto-build images based on tag, branch, and pull requests.
And will auto-comment the result to PR if the action trigger comes from PR.

That tags will be like [that](https://github.com/gocrane/crane/pkgs/container/crane%2Fdashboard).